cli/aeon/cmd/transport.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/maps"
+)
+
+// AeonTransport is an auxiliary type to constrain the list of stored value.
+type AeonTransport string
```

Comment:
It is just shorter and we don't loss anything since it is in a `aeon` package.
```suggestion
// Transport is an auxiliary type to constrain the list of stored value.
type Transport string
```